### PR TITLE
Corrected html2pdf pathes and pdf appearance.

### PR DESCRIPTION
**Pull request by alpinus4**
I changed path from:
```php
  require_once($GLOBALS['modules_dir'] . "/html2pdf/vendor/autoload.php");
```
to:
```php
  require_once($GLOBALS['modules_dir'] . "../../../../modules/html2pdf/vendor/autoload.php");
```

Fixes: https://github.com/LibreHealthIO/LibreEHR/issues/919

### CODE REVIEW

**Comment by alpinus4** [comment]
@teryhill

**Comment by teryhill** [comment]
I think the issue is actually in globals.php in the interface directory. It says $web_root and should actually say $webserver_root . I think that is the solution perhaps @aethelwulffe  can shed more light on it.

**Comment by teryhill** [comment]
@alpinus4  your fix will only work in this file and there are other files that print PDF's

**Comment by alpinus4** [comment]
@teryhill Fixed

**Comment by alpinus4** [comment]
Fixed @teryhill

**Comment by aethelwulffe** [comment]
$web_root is used all over the place, and is defined.
https://sourceforge.net/p/openemr/discussion/202505/thread/ba6075e2/

**Comment by alpinus4** [comment]
@aethelwulffe What do you mean? Is something wrong?

**Comment by alpinus4** [comment]
I fixed this: https://github.com/LibreHealthIO/LibreEHR/issues/929

Before: 



After:

**Comment by alpinus4** [comment]
@aethelwulffe @teryhill

File: interface/patient_file/report/custom_report.php [hunk 1]
```diff
@@ -488,21 +488,21 @@ function next_prev(action){
...
+     }
 ?>
-<h2><?php echo $facility['name'] ?></h2>
+<h2>&nbsp;&nbsp;&nbsp;<?php echo $facility['name'] ?></h2>
 <?php echo $facility['street'] ?><br>
 <?php echo $facility['city'] ?>, <?php echo $facility['state'] ?> <?php echo $facility['postal_code'] ?><br clear='all'>
 <?php echo $facility['phone'] ?><br>
 
-<a href="javascript:window.close();"><span class='title'><?php echo $titleres['fname'] . " " . $titleres['lname']; ?></span></a><br>
+<span class='title'><?php echo $titleres['fname'] . " " . $titleres['lname']; ?></span><br>
```
Comment (teryhill) [inline interface/patient_file/report/custom_report.php]: Why are you removing the hyperlink to close the screen?

Comment (alpinus4) [reply interface/patient_file/report/custom_report.php]: @pri2si17-1997 Said that I can do it.

Comment (alpinus4) [reply interface/patient_file/report/custom_report.php]: But I can restore it, if you want @teryhill

Comment (alpinus4) [reply interface/patient_file/report/custom_report.php]: Ok @pri2si17-1997 @teryhill 
I'll work on it soon

**Comment by alpinus4** [comment]
@pri2si17-1997 @teryhill I added `Close` link, which appears only in printable version.

**Comment by aethelwulffe** [comment]
Thanks for the update.  We can squash and merge this (after editing the commit message a bit).

I have no idea why $web_root doesn't work for you when it is defined and used in globals.php already, defined just like $scrdir is for the /libraries directory stuff.
Anywho, when changing the variable for a major directory alias, I suppose we need to test pretty extensively.  I will try to get my test and review done sometime tomorrow.

**Comment by aethelwulffe** [comment]
OK, this has a clean branch.  someone review please!

**Comment by aethelwulffe** [comment]
I fixed the conflicts with the other related PR that completely removes the logo from custom_report.  Merging...

**Pull request merged**